File: _challenges/6696c2ca763f0712c198dbf9_challenge.js
// NAMEEN:Double name
// NAMERU:Двойное имя

/*
INSTRUCTIONENSTART
Often Americans have two personal names.
Suppose the variable `name` stores the first name.
The variable `name` needs to get the first two names separated by a space.

There is a command that creates a variable `name` with the value `Ivan`:
```
let name = 'Ivan';
```
Write a command that changes the existing value of the `name` variable by concatenating it with the name 'Francis' with a space. The solution could be as follows:
```
name = name + ' ' + 'Francis';
```
INSTRUCTIONENEND
*/

/*
INSTRUCTIONRUSTART
![](https://course-qa-basics.s3.us-west-1.amazonaws.com/section-2-course-5-task-9.png)

Часто американцы имеют два личных имени.
Предположим, что в переменной `name` хранится первое имя.
Требуется в переменной `name` получить два первых имени через пробел.

Напишите команду, создающую переменную `name` со значением `Ivan`.

Запишите команду, которая изменяет имеющееся значение переменной `name`, соединяя его с именем 'Francis' через пробел.

Похожий пример:
```javascript
let name = 'Alice';
name = name + ' ' + 'Marie';
```
INSTRUCTIONRUEND
*/

/*
HINTENSTART
```javascript
let name = 'Ivan';
name = name + ' ' + 'Francis';
```
HINTENEND
*/

/*
HINTRUSTART
```javascript
let name = 'Ivan';
name = name + ' ' + 'Francis';
```
HINTRUEND
*/

/*
INITIALSOLUTIONSTART
let name = 'Ivan';
INITIALSOLUTIONEND
*/

// SOLUTIONSTART
let name = 'Ivan';
name = name + ' ' + 'Francis';
// SOLUTIONEND


// OPENTESTSSTART
it('The `name` variable contains a value Ivan Francis', () => {
    expect(name).eq('Ivan Francis');
});

it('The "name" is type of string', () => {
    expect(typeof name).eq('string');
});
// OPENTESTSEND

// HIDDENTESTSSTART

// HIDDENTESTSEND

File: _challenges/6696c2ca763f0712c198dbf9_challenge.py
// NAMEEN:How much money is left over from shopping?
// NAMERU:Сколько денег осталось после покупок?

/*
INSTRUCTIONENSTART
![](https://course-qa-basics.s3.us-west-1.amazonaws.com/section-2-course-5-task-12.png)

After the purchases were made in the store, the amount of money available decreased by $48.

Suppose that the variable `cash` stores some number.

Create a variable `cash` with value `200`.

Write a command that reduces the existing value of the `cash` variable by `48`.
INSTRUCTIONENEND
*/

/*
INSTRUCTIONRUSTART
![](https://course-qa-basics.s3.us-west-1.amazonaws.com/section-2-course-5-task-12.png)

После сделанных в магазине покупок количество имеющихся денег уменьшилось на 48 долларов.

Создайте переменную `cash` со значением `200`.

Запишите команду, которая уменьшает имеющееся значение переменной `cash` на `48`.
INSTRUCTIONRUEND
*/

/*
HINTENSTART
Similar example:
```python
cash = 100
cash = cash - 25
```
HINTENEND
*/

/*
HINTRUSTART
Похожий пример:
```python
cash = 100
cash = cash - 25
```
HINTRUEND
*/

/*
INITIALSOLUTIONSTART

INITIALSOLUTIONEND
*/

// SOLUTIONSTART
cash = 200
cash = cash - 48
// SOLUTIONEND

// OPENTESTSSTART
it('The "cash" variable should be defined', () => {
    expect(cash).not.undefined;
});

it('The "cash" variable should be a number', () => {
    expect(typeof cash).eq('number');
});

it('The "cash" variable should be equal to 152', () => {
    expect(cash).eq(152);
});
// OPENTESTSEND

// HIDDENTESTSSTART

// HIDDENTESTSEND
